### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by katrinleinweber**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

This PR is an on-going effort to provide the German translation. I'm using [DeepL.com/translator](https://www.deepl.com/translator) with some manual smoothing, and shortening of too verbose translations.

### Things to discuss (esp. with other German translators)

- [x] Use formal ~~or informal~~ `you`: `Sie` ~~or `Du`?. DeepL defaults to `Sie` which I'll stick to for now, to speed up the initial translation. Also, that might appeal more to those not already in FLOSS' field of gravity. 💡 other ideas:
  a) Split into a `Deutsch (Du)` and `Deutsch (Sie)` or `formal`/`informal` translation.
  b) Convert to `Du` at the end and explain that this more common in the FLOSS community in a note on the first page.
  c) Mix it (randomly).
  d) Shift it from `Sie` in the first articles to `Du` in the later ones.
  e) Use some kind of plugin that allows the use to select the variant, factor such phrases out and offer both.~~
- [x] "Maintainer" or "Betreuer"? DeepL's default is the latter, but I have mingled both expressions a bit.
- [x] "Open-Source-…" & "Pull-Request-…" for compound nouns, but "Open Source" and "Pull Request" when not compounded
- [x] italicize translated quotes (`pquote` and `>`) with `_` while retaining the original quote below it. That way, German readers can still find the original within the hyperlinked source by copy-pasting a snippet and searching for that.
- [x] ~~Renaming the files (translating the filenames) and adjusting their slugs seems logical, but~~ [the other translations retained the English filenames](https://github.com/github/opensource.guide/tree/master/_articles). Presuming that helps the main maintainers, let's mirror the other translators' behavior.
- [x] Prepare file translations on different branches in the fork, and merge them into this `de`-PR. Then: fix Travis errors here, because PRs in the fork aren't tested. Would that be worth a CI crash course to copy the upstream repo's pipeline?
- [x] change `Gemeinde`, because that would mean `municipality`

### CODE REVIEW

**Comment by katrinleinweber** [comment]
PS: If someone turned this bash command into a script that runs slower than the rate limit and replaces one line after the other line ;-)

**Comment by jens1o** [comment]
I think you should not rely on a translation software.

**Comment by katrinleinweber** [comment]
What do you mean with "rely"?

I argue that _utilising_ it to a) speed up the initial translation and b) experiment with the bash workflow described above is worth the risk and follow-up work of manually "smoothing out" an auto-translation.  Which (I think we agree here) is sometimes necessary. However, I experienced DeepL as pretty adept 

Do you maybe feel that the initial "speed up" part is eaten up by not "sometimes" but _often_ more "follow-up work"?

PS: We can of course also switch to German for such discussions among us translators  ;-)

**Comment by jens1o** [comment]
> Do you maybe feel that the initial "speed up" part is eaten up by not "sometimes" but often more "follow-up work"?

Yeah, that's the point.

> We can of course also switch to German for such discussions among us translators ;-)

I don't care about it, but we can do it if you'd like to.

**Comment by katrinleinweber** [comment]
Mir ist es auch egal :-D

**Comment by MikeMcQuaid** [comment]
@katrinleinweber I'm afraid I agree with @jens1o here: I think automatic translation tools are not a good fit for translating this project.

**Comment by katrinleinweber** [comment]
Are there any mistranslations so far? What I committed so far is slightly proof-read, _not_ the 100% auto-translation. You can compare [the latter](https://github.com/katrinleinweber/opensource.guide/pull/1/commits/51adbf8ad034a30394190e3ac2e072b27cbfbd83) with [the former](https://github.com/katrinleinweber/opensource.guide/pull/1/commits/e27271f5931e59a9fc53536125af98ba393ae9f7).

I feel that only a tool and the formal `you` have been criticized so far, not any specific translation result.

**Comment by MikeMcQuaid** [comment]
@katrinleinweber I don't speak German so cannot comment on any mistranslations. When this translation is complete I'll have a German friend or coworker review the results and comment.

**Comment by kulphol** [comment]
Dd

**Comment by katrinleinweber** [comment]
@MIKALFH83 OK, thank you. Will keep on translating in that style.

@kulphol I'm not sure what you mean, and I couldn't find a logical explanation for `Dd`. Do you mean _D_efinition of _d_one?

**Comment by katrinleinweber** [comment]
I am aware of the irony that appeasing Travis usually results in yet another ❌  but that is due to some other random hyperlink timing out during each integration. We'll git there eventually 🙄

File: _articles/de/getting-paid.md [hunk 1]
```diff
@@ -103,10 +103,10 @@ Wenn Sie kein existierendes Open-Source-Projekt haben, an dem Sie gerne arbeiten
...
+@hueniverse z.B., fand heraus, dass es finanzielle Gründe gab, die [Investition von Walmart in Open Source zu rechtfertigten.](https://www.infoworld.com/article/2608897/open-source-software/walmart-s-investment-in-open-source-isn-t-cheap.html). Und @jamesgpearce fand heraus, dass Facebooks Open-Source-Program [den Unterschied in der Personalakquise](https://opensource.com/business/14/10/head-of-open-source-facebook-oscon) machte:
 
 > Es ist eng mit unserer Hackerkultur und der Wahrnehmung unseres Unternehmens verbunden. Wir fragten unsere Angestellten: "Kanntet ihr das Open-Source-Software-Programm von Facebook?". Zwei Drittel sagten "Ja". Die Hälfte sagte, dass das Programm positiv zu ihrer Entscheidung beigetragen hat, für uns zu arbeiten. Das sind keine marginalen Zahlen, und ich hoffe, ein Trend, der sich fortsetzt.
-
+>
```
Comment (jens1o) [inline _articles/de/getting-paid.md]: why?

**Comment by Aslameng** [comment]
Hi

**Comment by Aslameng** [comment]
Here is a problem

**Comment by oanthailand** [comment]
a1de137a5afb12885b1a338709a636df927d3b9d

**Comment by dith99** [comment]
Nice

**Comment by katrinleinweber** [comment]
I'm not sure how to interpret the [two](https://github.com/github/opensource.guide/pull/577#issuecomment-387230233) [comments](https://github.com/github/opensource.guide/pull/577#issuecomment-387247884) above.

**Comment by andrittnt** [comment]
Good

**Comment by andrittnt** [comment]
Bagus

**Comment by petrk94** [comment]
@katrinleinweber 
Hello I would like to know whether you need help for the german translation?
Greets

**Comment by hdcorder147** [comment]
What do I need

**Comment by katrinleinweber** [comment]
@petrk94: Yes please, see https://github.com/katrinleinweber/opensource.guide/issues?q=is%3Aopen+sort%3Aupdated-desc

**Comment by jowagner** [comment]
It is difficult to review this PR from the "Files changed" tab. (1) To judge translation adequacy, a reviewer needs to see the English source next to each translation. (2) Realistically, a reviewer will only review less than 100 sentences. These sentences should be randomly sampled from all documents. Is there an easier way to do this than checking out both versions and a bit of scripting? 

Another question is what translation quality is necessary for the translation to be useful to the target audience. Most German readers have good English. I would think that the main reason why such readers might prefer a translation is that they can read it faster, with lower cognitive load, and to reduce the risk of misunderstandings. However, in the few sentences I read, the English source clearly shines through the word choices and sentence structure. This may be distracting and increase cognitive load.

**Comment by katrinleinweber** [comment]
> It is difficult to review this PR from the "Files changed" tab.

Here yes. That's why all translations [were reviewed in smaller chunks over in katrinleinweber/opensource.guide/pulls](https://github.com/katrinleinweber/opensource.guide/pulls?utf8=%E2%9C%93&q=is%3Apr) ;-)

> […] English source clearly shines through the word choices and sentence structure.

Agreed. This is a first draft. Shall improvement suggestions be PR'ed against `de` in the above fork, or after merging this against `master` here?

**Comment by katrinleinweber** [comment]
The `legal` translation was merged prematurely, sorry. Will be reverted & merged again.

**Comment by andrittnt** [comment]
Oppo naugat 7.1.1

**Comment by katrinleinweber** [comment]
About the [new task `Add a note to out-going links if they refer to English sources?`](https://github.com/github/opensource.guide/pull/577#issue-174154552): I don't think it is worth investing the effort, because in many cases, the URL will already contain English phrases.

File: _articles/de/metrics.md [hunk 2]
```diff
@@ -16,26 +16,26 @@ related:
   - best-practices
```
Comment (xDeaRxDeViLx) [inline _articles/de/metrics.md]: Beste - Praktiken

**Comment by jowagner** [comment]
I received an e-mail with reference to this PR 9h ago suggesting to translate "best-practices" with "Beste - Praktiken". (1) What's the hyphen doing here (both en and de)? (2) Why is the adjective capitalised? (3) That's a bad word-by-word translation. For inspiration, see for example https://dict.leo.org/englisch-deutsch/best%20practice

**Comment by mlinksva** [comment]
@jowagner it looks like the comment has been deleted, but it was on https://github.com/github/opensource.guide/pull/577/files#diff-27e291d6156c24caede48ab64171c2adR16 ... in that case `best-practices` is a slug/filename that should not be translated.

It seems that this repo (including this PR) has been getting an unusual number of spurious comments from new users. Mildly annoying, but I guess it is the product of lots of new users reading opensource.guide and coming to this repo. Success! :smile: Hopefully many more people and even some of the spurious commenters are learning lots by coming here.

**Comment by boonast69** [comment]
😶 😶 😶 😶

**Comment by katrinleinweber** [comment]
@mlinksva Thanks for clarifying :-)

@jowagner: Please don't rely on GitHub's notification emails, but on [the PRs in our translation fork](https://github.com/katrinleinweber/opensource.guide/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc). Review comments there can be addressed easiest. PRs that correct something about the translation are best sent against the `de` branch there. Danke sehr :-)

**Comment by MikeMcQuaid** [comment]
Hey @katrinleinweber! How's this going here? I've been out on paternity leave so have missed a bit.

**Comment by katrinleinweber** [comment]
Slowly but surely. I'll remove the `WIP` when this is ready.

**Comment by katrinleinweber** [comment]
- [ ] check escaping after `@Roshanjossey` in `_articles/de/how-to-contribute.md`

**Comment by mahdibakhtdar** [comment]
Hi,katrin.
Canyou help me about computer programmer?
در تاریخ ۱ سپتامبر ۲۰۱۸ ۶:۲۴، "Katrin Leinweber" <notifications@github.com>
نوشت:

**Comment by mahdibakhtdar** [comment]
How's the help me pleas?

**Comment by katrinleinweber** [comment]
@mahdibakhtdar: Please see https://try.github.io/, https://help.github.com/ and https://duckduckgo.com/.

**Comment by mahdibakhtdar** [comment]
Ok.

**Comment by mahdibakhtdar** [comment]
Thank you so much.

**Comment by katrinleinweber** [comment]
Ah, damn: Some non-optimal commit messages there, but since the build passed, I'm not going to squash, reword & force-push.

Almost done here :-) The last page's translation is being finalised in https://github.com/katrinleinweber/opensource.guide/pull/12.

**Comment by mahdibakhtdar** [comment]
Ok,Thank about your write

**Comment by mahdibakhtdar** [comment]
Ok,Thank about your write.

**Comment by katrinleinweber** [comment]
@jens1o & @Rubinum: Thanks so much for your help so far. I'll be pushing a few more corrections & improvements to `de` directly. Please feel free to also read through the current `.md` files in one go (which probably brings new perspectives compared to comparing diffs in the translation PRs), and suggest updates to phrases that do sound a bit strange still.

**Comment by Iosunlock** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> This PR is an on-going effort to provide the German translation. I'm using [DeepL.com/translator](https://www.deepl.com/translator) with some manual smoothing, and shortening of too verbose translations.
> 
> ### Things to discuss (esp. with other German translators)
> * [ ]  Use formal or informal `you`: `Sie` or `Du`? DeepL defaults to `Sie` which I'll stick to for now, to speed up the initial translation. Also, that might appeal more to those not already in FLOSS' field of gravity. 💡 other ideas:
>   a) Split into a `Deutsch (Du)` and `Deutsch (Sie)` or `formal`/`informal` translation.
>   b) Convert to `Du` at the end and explain that this more common in the FLOSS community in a note on the first page.
>   c) Mix it (randomly).
>   d) Shift it from `Sie` in the first articles to `Du` in the later ones.
>   e) Use some kind of plugin that allows the use to select the variant, factor such phrases out and offer both.
> * [x]  "Maintainer" or "Betreuer"? Again, using the latter due to DeepL's default.
> * [x]  "Open Source …" or "Open-Source-…" for compound nouns? Same as above ;-)
> * [ ]  translated quotes (`pquote` and `>`) should probably be italicized with `_` while retaining the original quote below it. That way, German readers can still find the original within the hyperlinked source by copy-pasting a snippet and searching for that.
> * [x]  Renaming the files (translating the filenames) and adjusting their slugs seems logical, but [the other translations retained the English filenames](https://github.com/github/opensource.guide/tree/master/_articles). Presuming that helps the main maintainers, let's mirror the other translators' behavior.
> * [x]  Prepare file translations on different branches in the fork, and merge them into this `de`-PR. Then: fix Travis errors here, because PRs in the fork aren't tested. Would that be worth a CI crash course to copy the upstream repo's pipeline?
> * [ ]  Add a note to out-going links if they refer to English sources? E.g. `(Englisch)`? This would tell German readers to be prepared to switch to English.
> * [ ]  change `Gemeinde`, because that would mean `municipality`

Unlock Activation

**Comment by Iosunlock** [comment]
Change attach file Master

**Comment by katrinleinweber** [comment]
I won't have time for reading through the German version once more to smooth it a bit until late November. The WP is not abandoned, though.

**Comment by karbazol666** [comment]
я не понимаю правила которые здесь установлены

File: _articles/de/starting-a-project.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+  </p>
+</aside>
+
+Manchmal vermeiden es Leute, eine README zu schreiben, weil sie das Gefühl haben, das Projekt sei unvollendet, oder weil sie keine Beiträge wollen. Aber auch dies sind gute Gründe, eine zu schreiben.
+
+Weitere Inspirationen zum Schreiben einer README finden Sie in @18F's ["Making READMEs Readable"](https://pages.18f.gov/open-source-guide/making-readmes-readable/) oder in @PurpleBooth's [README template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2).
+
+Wenn Sie eine README-Datei im Hauptordner Ihres Projektes anlegen, zeigt GitHub diese automatisch auf der Homepage des Repos an.
+
+### Ihre Beistragsrichtlinien aufschreiben
```
Comment (stefanmerkel) [inline _articles/de/starting-a-project.md]: Ihre **Beistragsrichtlinien** aufschreiben. _besser:_
Ihre **Beitragsrichtlinien** aufschreiben.

File: _articles/de/starting-a-project.md [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+order: 2
+image: /assets/images/cards/beginner.png
+related:
+  - finding
+  - building
+---
+
+## Was ist Open Source, und warum?
+
+Sie denken also über einen Sprung in die Open-Source-Welt nach? Herzlichen Glückwunsch, die Welt weiß Ihren Beitrag zu schätzen: Lassen Sie uns darüber sprechen, was Open Source ist und warum sich Leute dafür engagieren.
```
Comment (katrinleinweber) [inline _articles/de/starting-a-project.md]: Weiß sie das wirklch? Vielleicht lieber "und danke, dass Sie die Welt mit Ihrem Beitrag bereichern"?

**Comment by katrinleinweber** [comment]
Dear @MikeMcQuaid,

from my PoV this translation is now near-stable. Please let your ["German friend or coworker"](https://github.com/github/opensource.guide/pull/577#issuecomment-373293869) know. One last thing that may be left to do is:

- [ ] Add a note to out-going links if they refer to English sources? E.g. `(Englisch)`? This would tell German readers to be prepared to switch to English.

I'm happy if someone wants to do it by sending a PR against [`katrinleinweber:de`](https://github.com/katrinleinweber/opensource.guide/tree/de). I'd rather not do it myself, because I find that the browser's mechanism to preview URLs on hover makes it sufficiently clear that most external resources are in English.

Cheers :-)

**Comment by MikeMcQuaid** [comment]
@katrinleinweber Great work, thanks! Could you finish up the remaining test failures and then we should be good to go.

**Comment by MikeMcQuaid** [comment]
@katrinleinweber Still one more test failure! Note you can run `script/test` locally to reproduce.

**Comment by katrinleinweber** [comment]
I did and it yielded only a warning about [`Node.js'`](https://github.com/github/opensource.guide/pull/577/files#diff-62a661f16b4acbf4c751f7c08385f385R132). But that's how you spell a possessive form of a word that ends with an `s` :-/

**Comment by katrinleinweber** [comment]
Ah, there it is again :-) But I'll try to find a better workaround for this `s's` conundrum.

**Comment by MikeMcQuaid** [comment]
> Worth filing a bug to @retextjs' [retext-quotes](https://github.com/retextjs/retext-quotes/issues?utf8=%E2%9C%93&q=)?

Yeh 👍

File: de.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
+layout: index
+lang: de
```
Comment (MikeMcQuaid) [inline de.html]: Can this be modified like https://github.com/github/opensource.guide/commit/3dd15966d93ed26326a52fe008d7e570707a4f6e? Thanks!

**Comment by MikeMcQuaid** [comment]
Woohoo https://opensource.guide/de/ is live! Incredible work @katrinleinweber 👏!

**Comment by katrinleinweber** [comment]
You're welcome, and thanks for merging :-)

**Comment by katrinleinweber** [comment]
image

**Comment by katrinleinweber** [comment]
Follow-ups in #1118

**Comment by Burkay5441** [comment]
Türk'e

File: _articles/de/best-practices.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+Denken Sie daran, Ihre Dokumentation auf dem neuesten Stand zu halten. Wenn Sie dies nicht immer tun können, löschen Sie Veraltetes oder markieren es als solches, damit Mitwirkende wissen, dass Updates gerne angenommen werden.
+
+### Schreiben Sie Ihre Projektvision auf
+
+Schreiben Sie zunächst die Ziele Ihres Projekts auf. Fügen Sie sie Ihrer README hinzu oder erstellen Sie eine separate Datei namens VISION. Wenn es andere dafür nützliche Artefakte gibt (z.B. eine Projekt-Roadmap) machen Sie auch diese öffentlich.
+
+Eine klare, dokumentierte Vision hilft Ihnen, sich zu konzentrieren und den "Scope Creep" durch Beiträge anderer zu vermeiden.
+
+Zum Beispiel entdeckte @lord, dass eine Projektvision ihm herauszufinden half, in welche Anfragen er Zeit stecken sollte. Als frisch gebackener Maintainer bedauerte er, dass er sich nicht auf den Kern seines Projekts fokussiert hat, als er seine erste Feature-Anfrage für [Slate](https://github.com/lord/slate) erhielt.
```
Comment (newsize686) [inline _articles/de/best-practices.md:46]: _articles/building-community.md

**Comment by syedsadaat292-dev** [comment]
Han

**Comment by ashganhammad** [comment]
مين

**Pull request merged**